fmt/format.h

```diff
@@ -2138,8 +2138,10 @@ class ArgFormatterBase : public ArgVisitor<Impl, void> {
 
   // workaround MSVC two-phase lookup issue
   typedef internal::Arg Arg;
+  typedef Char          Char_Type;
+  typedef BasicWriter<Char> Writer_Type;
```

Comment:
`Char_Type` -> `CharType`, `Writer_Type` -> `WriterType` for consistency with other names.

Reply:
Should be fixed in 8915b2754397b5c4090267599d03cd31f1e323f0